### PR TITLE
Remove tasks and update readme for extension

### PR DESCRIPTION
**Pull request by agrojean-ledger**
The extension's README file provides information about what it can do as well : https://marketplace.visualstudio.com/items?itemName=LedgerHQ.ledger-dev-tools (or see the file directly in the github repo : https://github.com/LedgerHQ/ledger-vscode-extension)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,23 +15,15 @@ It will allow you, whether you are developing on macOS, Windows or Linux to quic
...
     * On Windows, install and launch [VcXsrv](https://sourceforge.net/projects/vcxsrv/) (make sure to configure it to disable access control).
-* Install [VScode](https://code.visualstudio.com/download)
+* Install [VScode](https://code.visualstudio.com/download) and add [Ledger's extension](https://marketplace.visualstudio.com/items?itemName=LedgerHQ.ledger-dev-tools).
 * Open a terminal and clone `app-boilerplate` with `git clone git@github.com:LedgerHQ/app-boilerplate.git`.
 * Open the `app-boilerplate` folder with VSCode.
-* Open the vscode tasks with  `ctrl + shift + b` (`command + shift + b` on a Mac) and run the following actions :
-    * Pull and run the [ledger-app-dev-tools](https://github.com/LedgerHQ/ledger-app-builder/pkgs/container/ledger-app-builder%2Fledger-app-dev-tools) docker image by selecting `Run dev-tools image`.
-    * Build the for the device model of your choice with `Build app`.
-    * Test your binary on [Speculos](https://github.com/LedgerHQ/speculos) with `Run Speculos`.
+* Use the extension's sidebar menu or open the tasks menu with `ctrl + shift + b` (`command + shift + b` on a Mac) to conveniently execute actions :
```
Comment (tdejoigny-ledger) [inline README.md:21]: add `Ledger `before `extension`

File: README.md [hunk 2]
```diff
@@ -15,23 +15,15 @@ It will allow you, whether you are developing on macOS, Windows or Linux to quic
...
+    * Build the app for the device model of your choice with `Build`.
+    * Test your binary on [Speculos](https://github.com/LedgerHQ/speculos) with `Run with Speculos`.
+    * ...
```
Comment (tdejoigny-ledger) [inline README.md:24]: should we also list main other actions ? at least sideload

File: README.md [hunk 3]
```diff
@@ -147,21 +139,20 @@ The boilerplate app comes with functional tests implemented with Ledger's [Ragge
...
-* `Install tests requirements` to be executed once after you have run the dev-tools image, followed by,
-* `Run functional tests` and select the model you just built the app for.
+* Use `Run tests`.
 
 Or simply run the app on the Speculos emulator :
 
-* `Test app with Speculos` and select the correct device model.
+* `Run with Speculos`.
 
 ### Linux (Ubuntu)
```
Comment (tdejoigny-ledger) [inline README.md:158]: This part should be updated "[VSCode integration](#with-vscode)" --> extension

Comment (tdejoigny-ledger) [reply README.md:158]: could be the good time to include this [PR](https://github.com/LedgerHQ/app-boilerplate/pull/90/files)

File: README.md (hunk 1 above)
Comment (agrojean-ledger) [reply README.md:21]: Ok

File: README.md (hunk 2 above)
Comment (agrojean-ledger) [reply README.md:24]: Done.

File: README.md (hunk 3 above)
Comment (agrojean-ledger) [reply README.md:158]: Done. And I added the extension in `.vscoe/extensions.json`.

**Review by tdejoigny-ledger — APPROVED** [review]

**Pull request merged**